crates/proto/src/generated/blockchain.rs

```diff
@@ -62,20 +62,21 @@ pub struct BlockHeader {
     /// A commitment to all transaction kernels supported by this block.
     #[prost(message, optional, tag = "10")]
     pub tx_kernel_commitment: ::core::option::Option<super::primitives::Digest>,
-    /// Fee parameters for block processing.
-    #[prost(message, optional, tag = "11")]
-    pub fee_parameters: ::core::option::Option<FeeParameters>,
     /// The time when the block was created.
-    #[prost(fixed32, tag = "12")]
+    #[prost(fixed32, tag = "11")]
```

Comment:
From experience, this shouldn't change after once used. The error message used to funny.